files/qsub.pl

```diff
@@ -583,28 +583,31 @@ sub parse_script
         }
     };
 
-    # handle queues
-    if ($orig_argsh{'q'} || $set{'q'}) {
-        warn "Please do not set a queue (\'-q\' option or \'#PBS -q\' directive).\n" .
-            "Right now it sets only the default walltime based on the queue,\n" .
-            "So please request explicitly the desired walltime.\n";
-
-    }
+    # handle queues and special queues (as a partition)
     my $userqueue = $destination || $set{'q'};
     if ($userqueue) {
+        if ($userqueue eq "special" ) {
+            push(@cmd, '--partition', "$ENV{'VSC_INSTITUTE_CLUSTER'}_$userqueue") if $ENV{'VSC_INSTITUTE_CLUSTER'};
```

Comment:
at least warn or even die when the `VSC_INSTITUTE_CLUSTER` is not set. and fyi, the quotes are not needed in the hashes, `$ENV{VSC_INSTITUTE_CLUSTER}` works fine too

Reply:
done